src/views/Nft/market/Profile/hooks/useGetUserNfts.tsx

```diff
@@ -0,0 +1,35 @@
+import { useEffect, useState } from 'react'
+import { useSelector } from 'react-redux'
+import { useAppDispatch } from 'state'
+import Nfts from 'config/constants/nfts'
+import { State } from 'state/types'
+import { fetchWalletNfts } from 'state/collectibles'
+import { fetchUserNfts } from 'state/nftMarket/reducer'
+
+// We need to fetch collectibles for non-connected accounts, hence this hook accepts an account string.
+const useGetOwnedNfts = (account: string) => {
```

Comment:
VERY much in progress hook to return user nft data when visiting the new profile page